### PR TITLE
ALFREDAPI-461 reintroduce xenit maven repo

### PR DESCRIPTION
**Pull request by RVanhuysseXenit**
Fixes https://xenitsupport.jira.com/browse/ALFREDAPI-461

- [ ] Is [CHANGELOG.md](https://github.com/xenit-eu/alfred-api/blob/master/CHANGELOG.md) extended?
- [ ] Does this PR avoid breaking the API? 
    Breaking changes include adding, changing or removing endpoints and/or JSON objects used in requests and responses.
- [ ] Does the PR comply to REST HTTP result codes policy outlined in the [developer guide](https://github.com/xenit-eu/alfred-api/blob/master/developer-documentation)?
- [ ] Is error handling done through a method annotated with `@ExceptionHandler` in the webscript classes?
- [ ] Does the PR follow our [coding styleguide and other active procedures](https://xenitsupport.jira.com/wiki/spaces/XEN/pages/624558081/XeniT+Enhancement+Proposals+XEP)?
- [ ] Is usage of `this.` prefix avoided?

See [README.md](./README.md) for full explanation.

### CODE REVIEW

**Review by WimCrols — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 
 # Alfred API - Changelog
 
+## 2.7.1 (Unreleased)
```
Comment (WimCrols) [inline CHANGELOG.md:4]: Can you put your change under 2.7.0?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
 
 # Alfred API - Changelog
 
+## 2.7.0 (unreleased)
 
-
-## 2.7.0 (2020-10-01)
+### Changed
+* [ALFREDAPI-461](https://xenitsupport.jira.com/browse/ALFREDAPI-418): Reintroduce xenit maven repo definition
```
Comment (koenvangeerteruy) [inline CHANGELOG.md:7]: @RVanhuysseXenit Link is not pointing to the corresponding ticket.

Comment (RVanhuysseXenit) [reply CHANGELOG.md]: good spot. fixed it

**Review by WimCrols — APPROVED** [review]

**Comment by WimCrols** [comment]
BTW please rebase/squash those commits. 1-2 commits should be sufficient for this. @RVanhuysseXenit

**Review by koenvangeerteruy — APPROVED** [review]

**Pull request merged**